bin/dcps_short_tests.lst

```diff
@@ -54,6 +54,13 @@ tests/DCPS/DpShutdown/run_test.pl tcp: !DCPS_MIN !OPENDDS_SAFETY_PROFILE !DDS_NO
 tests/DCPS/DpShutdown/run_test.pl shmem: !DCPS_MIN !OPENDDS_SAFETY_PROFILE !DDS_NO_OWNERSHIP_PROFILE !NO_SHMEM
 tests/DCPS/DpShutdown/run_test.pl udp: !DCPS_MIN !OPENDDS_SAFETY_PROFILE !DDS_NO_OWNERSHIP_PROFILE
 tests/DCPS/DpShutdown/run_test.pl multicast: !DCPS_MIN !OPENDDS_SAFETY_PROFILE !DDS_NO_OWNERSHIP_PROFILE !NO_MCAST
+
+tests/DCPS/ContainsEntity/run_test.pl: RTPS
+tests/DCPS/ContainsEntity/run_test.pl tcp: !DCPS_MIN !OPENDDS_SAFETY_PROFILE !DDS_NO_OWNERSHIP_PROFILE
+tests/DCPS/ContainsEntity/run_test.pl shmem: !DCPS_MIN !OPENDDS_SAFETY_PROFILE !DDS_NO_OWNERSHIP_PROFILE !NO_SHMEM
+tests/DCPS/ContainsEntity/run_test.pl udp: !DCPS_MIN !OPENDDS_SAFETY_PROFILE !DDS_NO_OWNERSHIP_PROFILE
+tests/DCPS/ContainsEntity/run_test.pl multicast: !DCPS_MIN !OPENDDS_SAFETY_PROFILE !DDS_NO_OWNERSHIP_PROFILE !NO_MCAST
```

Comment:
Does the test need to run 5 times with different transports?  It's testing behavior at the DCPS layer, seems like the transport type shouldn't matter.

Reply:
Just running it with RTPS only as part of the test suite